model/model/v1alpha/model.proto

```diff
@@ -1737,3 +1738,96 @@ message UndeployOrganizationModelAdminRequest {
 
 // UndeployOrganizationModelAdminResponse represents a response for a undeployed model
 message UndeployOrganizationModelAdminResponse {}
+
+// ModelTrigger contains information about trigger of models.
+message ModelTrigger {
+  // TriggerStatus defines the status of a model trigger
+  enum TriggerStatus {
+    // Unspecified
+    TRIGGER_STATUS_UNSPECIFIED = 0;
+    // model trigger in progress
+    TRIGGER_STATUS_PROCESSING = 1;
+    // model trigger succeeded
+    TRIGGER_STATUS_COMPLETED = 2;
+    // model trigger failed
+    TRIGGER_STATUS_FAILED = 3;
+    // model trigger is waiting to be executed
+    TRIGGER_STATUS_QUEUED = 4;
+  }
+
+  // Namespace ID
+  string namespace_id = 1 [(google.api.field_behavior) = REQUIRED];
+  // Model UUID.
+  string model_uid = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger UUID.
+  string trigger_uid = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model trigger status
+  TriggerStatus status = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model visibility.
+  Model.Visibility visibility = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger source
+  string source = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger start time
+  google.protobuf.Timestamp start_time = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger total duration in milliseconds
+  optional google.protobuf.Duration total_duration = 8 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger end time
+  optional google.protobuf.Timestamp end_time = 9 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger requesters
+  string runner = 10 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
Is this the requester ID? UUID? Let's be more specific. What's the difference between this field and `namespace_id`? We should clarify that through naming / documentation, too.

```suggestion
  // Trigger requester ID.
  string requester_id = 10 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Reply:
I'm not sure about the difference between requester_id and namespace_id. Are they always the same in a model trigger request?

Reply:
checked again in FE design and documents, there is no namespace id field.
will remove it later